server/app/views/admin/questions/QuestionsListView.java

```diff
@@ -105,49 +111,31 @@ private DivTag renderSummary(ActiveAndDraftQuestions activeAndDraftQuestions) {
   }
 
-  /** Renders the full table. */
-  private Pair<TableTag, ImmutableList<Modal>> renderQuestionTable(
+  /**
+   * @return a pair. First element is list of DOM elements, each representing a row for a particular
+   *     question. Second element is collection of all modals that can be invoked from question
+   *     rows.
+   */
+  private Pair<ImmutableList<DomContent>, ImmutableList<Modal>> renderAllQuestionRows(
       ActiveAndDraftQuestions activeAndDraftQuestions, Http.Request request) {
-    ImmutableList<Pair<TrTag, ImmutableList<Modal>>> tableRowAndModals =
-        activeAndDraftQuestions.getQuestionNames().stream()
-            .map(
-                (questionName) ->
-                    renderQuestionTableRow(questionName, activeAndDraftQuestions, request))
-            .collect(ImmutableList.toImmutableList());
-    TableTag tableTag =
-        table()
-            .withClasses(Styles.BORDER, Styles.BORDER_GRAY_300, Styles.SHADOW_MD, Styles.W_FULL)
-            .with(renderQuestionTableHeaderRow())
-            .with(tbody(each(tableRowAndModals, (tableRowAndModal) -> tableRowAndModal.getLeft())));
-    ImmutableList<Modal> modals =
-        tableRowAndModals.stream()
-            .map(Pair::getRight)
-            .flatMap(ImmutableList::stream)
-            .collect(ImmutableList.toImmutableList());
-    return Pair.of(tableTag, modals);
-  }
-
-  /** Render the question table header row. */
-  private TheadTag renderQuestionTableHeaderRow() {
-    return thead(
-        tr().withClasses(Styles.BORDER_B, Styles.BG_GRAY_200, Styles.TEXT_LEFT)
-            .with(th("Info").withClasses(BaseStyles.TABLE_CELL_STYLES, Styles.W_2_12))
-            .with(th("Question text").withClasses(BaseStyles.TABLE_CELL_STYLES, Styles.W_3_12))
-            .with(
-                th("Supported languages").withClasses(BaseStyles.TABLE_CELL_STYLES, Styles.W_2_12))
-            .with(
-                th("Referencing programs").withClasses(BaseStyles.TABLE_CELL_STYLES, Styles.W_2_12))
-            .with(
-                th("Actions")
-                    .withClasses(BaseStyles.TABLE_CELL_STYLES, Styles.PL_8, Styles.W_2_12)));
+    ImmutableList.Builder<DomContent> rows = ImmutableList.builder();
+    ImmutableList.Builder<Modal> modals = ImmutableList.builder();
+    for (String name : activeAndDraftQuestions.getQuestionNames()) {
+      Pair<DivTag, ImmutableList<Modal>> rowAndModals =
+          renderQuestionRow(name, activeAndDraftQuestions, request);
+      rows.add(rowAndModals.getLeft());
+      modals.addAll(rowAndModals.getRight());
+    }
+    return Pair.of(rows.build(), modals.build());
   }
 
   /**
-   * Display this as a table row with all fields.
+   * Renders a row in the question list. The row contains question text, help text, active and draft
+   * versions, list of programs questions is used in and buttons to edit the question.
    *
    * <p>One of {@code activeDefinition} and {@code draftDefinition} must be specified.
```

Comment:
I think this comment is no longer relevant (and wasn't relevant prior to your PR either).

Reply:
Specifically, I mean the "One of activeDefinition and draftDefinition must be specified" line.

Reply:
Done.